packages/@azure/servicebus/data-plane/package.json

```diff
@@ -31,6 +33,7 @@
     "tslib": "^1.9.3"
   },
   "devDependencies": {
+    "@microsoft/api-extractor": "^6.3.0",
```

Comment:
Will this go away when we rebase against latest master?

Reply:
This is not an addition. It just got moved up. I think @HarshaNalluru has a VS Code extension that orders the dependencies alphabetically.

Reply:
Yes, alphabetical order.